cmd/root.go

```diff
@@ -151,7 +151,6 @@ func rootCmd(o *options.Options) error {
 		for checkName := range enabledChecks {
 			fmt.Fprintf(os.Stderr, "Finished [%s]\n", checkName)
 		}
-		fmt.Println("\nRESULTS\n-------")
```

Comment:
This seems like it wasn't added back

Reply:
It wasn't, it will print as before to console by default tho. The difference is when it's printed to a file, the file would contain this "Results" header. Should we revert this anyway?

Reply:
Reverted.